### PR TITLE
Add Reset()

### PR DESCRIPTION
**Pull request by 6543**
Add Reset() to reset locale store

### CODE REVIEW

**Comment by 6543** [comment]
@unknwon this make it possible to reset i18n on runtime & coverage got from 47.4% -> 51.9% :tada:

**Comment by 6543** [comment]
@unknwon do you have any questions about it?

**Comment by 6543** [comment]
@unknwon should I squash the fix in ...?

**Comment by 6543** [comment]
> @unknwon should I squash (second commit) the fix in ...?

squashed ...

**Comment by 6543** [comment]
@unknwon any change you have time for a review/response?

File: i18n.go [hunk 1]
```diff
@@ -174,6 +174,11 @@ func SetMessageWithDesc(lang, langDesc string, localeFile interface{}, otherLoca
 	return err
 }
 
+// Flush reset locale store
```
Comment (unknwon) [inline i18n.go:177]:
```suggestion
// Flush resets locale store.
```

**Review by unknwon — APPROVED** [review]
Sorry about waiting and thanks for the PR :)

**Comment by unknwon** [comment]
I can't apply suggestion for you, would you mind applying? Also no need to squash commits within the PR, will squash when merge.

**Comment by 6543** [comment]
@unknwon thanks :)

I did rename it to `Reset` as **lunny** also think it's more accurate  
-> ref: https://github.com/go-gitea/gitea/pull/15073#pullrequestreview-617013130

**Comment by 6543** [comment]
hope this is fine now :sweat_smile:

**Comment by unknwon** [comment]
Thanks again :)

**Pull request merged**